### DISCUSSION TITLE
SPI Driver [SPI_MSC313] have problem on FIFO transfer

### DISCUSSION THREAD

**Discussion by AhmadPanogari** [linux-chenxing/linux-chenxing.org#78] [General]
I'm currently implementing DRM for ILI9488 (3.5 Inch LCD) with SPI interface

and found an error message like in the picture.

I will attach the driver code
![image](https://user-images.githubusercontent.com/17040103/213686279-de4526fd-241f-4a42-b581-123e94af0582.png)

**Comment by AhmadPanogari** [comment]
Here the Driver Code

https://gist.github.com/AhmadPanogari/74a1543c6374e136175a23574eb34bbb

**Comment by fifteenhex** [comment]
This means the DMA transfer either didn't trigger or timed out. I found some issues with certain sizes causing that. I will take a look.

**Reply by AhmadPanogari** [reply]
Thankyou :)

**Comment by fifteenhex** [comment]
Please try pulling mstar_v6_2_rebase and see if it helps. There is still an issue with the spi not triggering every so often but the DMA side knows how to timeout now and the error shouldn't keep repeating.
